### PR TITLE
Add v8 alpha.1 changelog and fix-up package README

### LINKED ISSUE #1293: Improve NuGet package README
The [README for the NuGet package](https://www.nuget.org/packages/Polly/7.2.4#readme-body-tab) has a few broken bits of HTML due to CSP restrictions and HTML rendering differences compared to when rendered in GitHub which result in a few bits looking broken.

We should either:
1. Change the README to be compatible with rendering in NuGet.org;
2. Create a separate README that's more bare-bones and links back to the main repo README.

I think option 2 is better as it doesn't send us down the path of the lowest common denominator.

Other thoughts?

### PR DESCRIPTION
**Pull request by martincostello**
- Update the CHANGELOG for `8.0.0-alpha.1`.
- Add dedicated NuGet package README that is much simpler and delegates most content to the repository.
 Resolves #1293.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+
+- The first public preview of our [new high-performance core API and extensions](https://github.com/App-vNext/Polly/issues/1048) - Thanks to:
+  - [@andrey-noskov](https://github.com/andrey-noskov)
+  - [@joelhulen](https://github.com/joelhulen)
+  - [@juraj-blazek](https://github.com/juraj-blazek)
+  - [@geeknoid](https://github.com/geeknoid)
+  - [@martincostello](https://github.com/martincostello)
+  - [@martintmk](https://github.com/martintmk)
+  - [@SimonCropp](https://github.com/SimonCropp)
+  - [@terrajobst](https://github.com/terrajobst)
```
Comment (martincostello) [inline CHANGELOG.md:11]: Let me know if I forgot anyone!

Comment (martintmk) [reply CHANGELOG.md:11]: Let's also mention:

- @tekian: he helper me shape the API surface
- @laura-mi: she was influential behind the resilience API that was precursor to V8
- @adamnova: he provided great internal feedback

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
+## 8.0.0-alpha.1
+
+- The first public preview of our [new high-performance core API and extensions](https://github.com/App-vNext/Polly/issues/1048) - Thanks to:
```
Comment (martintmk) [inline CHANGELOG.md:3]: Can we also reference the samples directory (to be merged) and this readme that explains the architecture?

https://github.com/App-vNext/Polly/blob/main/src/Polly.Core/README.md

Comment (martincostello) [reply CHANGELOG.md:3]: I'll add the samples once it's merged (or you can rebase onto this once merged and add it yourself).

**Pull request merged**